File: Rally-BackEnd/backend/src/main/java/org/rally/backend/userprofilearm/model/dto/HidePostDTO.java
package org.rally.backend.userprofilearm.model.dto;

public class HidePostDTO {

    public String postType;
    public Integer hidePostId;
    public Integer userId;

    public HidePostDTO(String postType, Integer hidePostId, Integer userId) {
        this.postType = postType;
        this.hidePostId = hidePostId;
        this.userId = userId;
    }

    public HidePostDTO() {
    }

    public String getPostType() {
        return postType;
    }

    public void setPostType(String postType) {
        this.postType = postType;
    }

    public Integer getHidePostId() {
        return hidePostId;
    }

    public void setHidePostId(Integer hidePostId) {
        this.hidePostId = hidePostId;
    }

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }
}

File: Rally-FrontEnd/src/app/user-profile-arm/models/dto/HidePostDTO.ts
export class HidePostDTO {

	postType: string;
    hidePostId: number;
    userId: number;
	
	constructor(postType: string, hidePostId: number, userId: number){
		this.postType = postType;
		this.hidePostId = hidePostId;
        this.userId = userId;
	}
}